docs/docs/using-pants/using-pants-in-ci.mdx

```diff
@@ -44,7 +44,10 @@ limit_mb=$2
 size_mb=$(du -m -d0 "${path}" | cut -f 1)
```

Comment:
I noticed that the conversion script accidentally dedented this codeblock. I suspect codeblocks inside of admonitions were effected.

I'll file an issue and fix in another PR